test/cli/main.js

```diff
@@ -280,6 +314,94 @@ QUnit.module( "CLI Main", () => {
 				} );
 			}
 		} );
+
+		QUnit.test( "errors if notrycatch is used and a rejection occurs in a hook", async assert => {
+			try {
+				await execute( "qunit notrycatch/returns-rejection-in-hook.js" );
+			} catch ( e ) {
+				assert.pushResult( {
+
+					// only in stdout due to using `console.log` in manual `unhandledRejection` handler
+					result: e.stdout.indexOf( "Unhandled Rejection: bad things happen sometimes" ) > -1,
+					actual: e.stdout + "\n" + e.stderr
+				} );
+			}
+		} );
+	} );
+
+	QUnit.test( "config.module", async assert => {
+		const command = "qunit config-module.js";
+		const execution = await execute( command );
+
+		assert.equal( execution.code, 0 );
+		assert.equal( execution.stderr, "" );
+		assert.equal( execution.stdout, expectedOutput[ command ] );
+	} );
+
+	QUnit.test( "config.testTimeout", async assert => {
+		const command = "qunit config-testTimeout.js";
+
+		try {
+			await execute( command );
+		} catch ( e ) {
+			assert.equal( e.code, 1 );
+			assert.equal( e.stderr, "" );
+			assert.equal( e.stdout, expectedOutput[ command ] );
+		}
+	} );
+
+	QUnit.module( "noglobals", () => {
+		QUnit.test( "add global variable", async assert => {
+			try {
+				await execute( "qunit noglobals/add-global.js" );
+			} catch ( e ) {
+				assert.pushResult( {
+					result: e.stdout.indexOf( "message: Introduced global variable(s): dummyGlobal" ) > -1,
+					actual: e.stdout + "\n" + e.stderr
+				} );
+			}
+		} );
+
+		QUnit.test( "remove global variable", async assert => {
+			try {
+				await execute( "qunit noglobals/remove-global.js" );
+			} catch ( e ) {
+				assert.pushResult( {
+					result: e.stdout.indexOf( "message: Deleted global variable(s): dummyGlobal" ) > -1,
+					actual: e.stdout + "\n" + e.stderr
+				} );
+			}
+		} );
+
+		QUnit.test( "forgive global variable", async assert => {
```

Comment:
Took me a minute to realize why this one should pass since it isn't configured or CLI-option'ed to behave differnet, but it's that `qunit-` internals are allowed to be global indeed. Might be worth adding to the test name, e.g. "forgive qunit DOM globals" or some such.